shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -233,6 +236,11 @@ - (void)resetChannels {
 // Channels get a reference to the engine, and therefore need manual
 // cleanup for proper collection.
 - (void)setupChannels {
+  [_binaryMessenger setMessageHandlerOnChannel:@"flutter/isolate"
```

Comment:
Maybe add a comment on when you would expect to receive this call from the other side.